### PR TITLE
Update Indonesian Language

### PR DESCRIPTION
**Pull request by exodius48**
Update Indonesian Language as per v1.2.1 update

### CODE REVIEW

File: res/values-in/strings.xml [hunk 1]
```diff
@@ -50,29 +50,29 @@
...
     <string name="force_stop_15min">15 menit</string>
     <string name="force_stop_30min">30 menit</string>
     <string name="force_stop_1hour">1 jam</string>
-    <string name="destroy_processes">Don\'t keep processes</string>
-    <string name="destroy_processes_summary">Force stop applications after the user destroys all activities. Recommend off.</string>
-    <string name="no_license">You don\'t have the license to modify this setting, please donate at least 3 usd, then contact the author to get one.</string>
-    <string name="no_valid_license">This license is for \"%1$s\", you have no permission to use it. Please click \"%2$s\" for a new one.</string>
-    <string name="apply">apply</string>
+    <string name="destroy_processes">Jangan simpan proses</string>
+    <string name="destroy_processes_summary">Paksa berhenti aplikasi setelah pengguna meniadakan semua aktifitas. disarankan off.</string>
```
Comment (liudongmiao) [inline res/values-in/strings.xml]: Please check for "disarankan off", I think it should be "disarankan off".

File: res/values-in/strings.xml [hunk 2]
```diff
@@ -50,29 +50,29 @@
     <!-- advanced settings -->
     <string name="screen_off">Layar mati</string>
     <string name="force_stop_label">Paksa berhenti</string>
-    <string name="force_stop_summary">Nach %1$s ohne Aktivität</string>
-    <string name="force_stop_never">Never (Recommend)</string>
+    <string name="force_stop_summary">Setelah %1$s tanpa aktivitas</string>
```
Comment (liudongmiao) [inline res/values-in/strings.xml]: Sorry, I copy it from `values-de`, not `values-in`.
And I suggest it to "`Setelah tidak aktif selama %1$s`", which is the same in `Settings`.
https://github.com/android/platform_packages_apps_settings/blob/master/res/values-in/strings.xml#L831

**Comment by liudongmiao** [comment]
please help to update the translation. thanks.

**Comment by exodius48** [comment]
I'll try. :)

**Pull request merged**